store/models.py

```diff
@@ -40,6 +40,7 @@ class Order(models.Model):
     store = models.ForeignKey(Store, on_delete=models.CASCADE)
     products = models.ManyToManyField(Product, through="OrderItem")
     total_cost = models.DecimalField(max_digits=10, decimal_places=2, validators=[MinValueValidator(1.00)])
+    paid_on = models.DateTimeField(auto_now_add=True)
```

Comment:
The auto_now_add=True argument would not be ideal here.
This is because without the stripe integration, the store owner might need to manual mark the order as paid no?
And in the case where stripe is integrated, paid_on will be populated by the triggered webhook?

Reply:
Based on the existing workflow, if a stripe API key doesn't exist, it will create orders without the stripe payment process.

I think we're both on the same page that it does add a time stamp whether stripe is used or not. I'm quite sure by the use case you're talking about. I was under the impression that store admins could mark it paid manually in their admin panel if needed.

Is that not what you're talking about?

Reply:
O, 🤦‍♂️. I see what you mean. Yes, I'd have to assign the time stamp in the webhook but leave the value blank/null if they didn't go through the stripe payment process.

I'll go ahead and fix this.